opal/mca/if/bsdx_ipv6/configure.m4

```diff
@@ -29,20 +30,20 @@ AC_DEFUN([MCA_opal_if_bsdx_ipv6_CONFIG], [
     # happy.  I.e., this:
     #if defined( __NetBSD__) || defined(__OpenBSD__) || defined(__FreeBSD__) || \
     #             defined(__386BSD__) || defined(__bsdi__) || defined(__APPLE__)
-    AC_MSG_CHECKING([struct sockaddr])
-    AS_IF([test "$opal_found_sockaddr" = "yes"],
-          [AC_MSG_RESULT([yes (cached)])
+    AC_MSG_CHECKING([struct sockaddr_in])
```

Comment:
Why would we care is `struct sockaddr_in` is defined in an IPV6 bound module ?

Reply:
I think we were using the check for `sockaddr_in` as a proxy for also checking for IPv6 support / structures.  I.e., back when this was written, I think that if you have IPv4 support, you also had IPv6 support.  It looks like it was replacing C #if's like this:

```
    # If we found struct sockaddr and we're on any of the BSDs, we're                           
    # happy.  I.e., this:                                                                       
    #if defined( __NetBSD__) || defined(__OpenBSD__) || defined(__FreeBSD__) || \               
    #             defined(__386BSD__) || defined(__bsdi__) || defined(__APPLE__)                
```

Reply:
@bosilca Can you comment on this? ^^

Reply:
There is a difference between `struct sockaddr` (the umbrella struct for all sockaddr) and a specific protocol support such as `struct sockaddr_in` (specific for IPv4). But, even if these two checks (`sockaddr` and `sockaddr_in`) are morally equivalent on BSD, it seems like a stretch to infer that IPv4 support automatically grants IPv6 (again, even if in this particular case, BSD-like OS, it is the case). Personally, I prefer the old inference, `struct sockaddr` implies `sockaddr_in6` on BSD.